### ISSUE TITLE
Refactor media post

### ISSUE THREAD

**Issue by wesleya** [wesleya/hashtagaway-spark#1]
* if you use the same hashtag in your message more than once, it shows the hashtag multiple times. it should only show unique hashtags on the success page

once this is done we can 

* add tests to cover the entire process. right now there are some parts that are hard to test. (separate ticket for this?) 
* update  to use amazon web storage instead of digital ocean droplet space (separate ticket for this?)
